Salesforce/manifest.json

```
@@ -0,0 +1,13 @@
+{
+    "configuration": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "properties": {},
+        "title": "Salesforce module Configuration",
+        "type": "object"
+    },
+    "description": "Salesforce provides users comprehensive tools to manage customer data, automate processes, analyze data and insights, and create personalized customer experiences. Salesforce also offers a variety of solutions for customer service, marketing automation, commerce, app development, and more",
+    "name": "Salesforce",
+    "uuid": "626c6110-0a7a-11ee-be56-0242ac120002",
+    "slug": "salesforce",
+    "version": "1.22"
```

We start module version with `1.0.0` :D
```suggestion
    "version": "1.0.0"
```